controllers/greptimedbcluster/controller.go

```diff
@@ -79,6 +80,7 @@ func (r *Reconciler) SetupWithManager(mgr ctrl.Manager) error {
 		Owns(&corev1.Service{}).
 		Owns(&appsv1.StatefulSet{}).
 		Owns(&appsv1.Deployment{}).
+		Owns(&v1alpha1.GreptimeDBStandalone{}).
```

Comment:
Why does the greptimedbcluster controller need to add `v1alpha1.GreptimeDBStandalone`?

Reply:
Because it will create the standalone to monitor the cluster.